accountsplus/settings.py

```diff
@@ -0,0 +1,62 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.apps import apps
+
+
+# Default values
+LOCKOUT_TEMPLATE = 'accounts/locked_out.html'
+
+def get_setting(setting_str, is_required, default_value=None):
+    try:
+        return getattr(settings, setting_str)
+    except AttributeError:
+        if is_required:
+            raise ImproperlyConfigured('Setting {0} is not configured in your settings module'.format(setting_str))
```

Comment:
OK! but still we would need this kind of formatting when we have multiple values to be formatted within the list, right?